### ISSUE TITLE
change default resolution for non-benchmark regression tests to c96+1deg ocean/ice

### ISSUE THREAD

**Issue by DeniseWorthen** [ufs-community/ufs-s2s-model#190]
**Description**
Non-benchmark regression tests should use c96+1deg ocean/ice. 

**Solution**
Create a set of tests which replicate the current c96+1/4 deg ocean/ice tests. 

**Alternatives**

**Additional context**

**Comment by JessicaMeixner-NOAA** [comment]
Would it also help to have the "benchmark test" just be the test with waves and remove the non-waves benchmark test?

**Comment by JessicaMeixner-NOAA** [comment]
Would changing the PET counts help to minimize the runtimes?

**Comment by DeniseWorthen** [comment]
OK, sounds good, I'll leave it at 1 d for now.

For PEs, I suspect I'm over-resourcing the C96 and C192 tests (based on what Phil is using) and under-resourcing the C384. But I'd prefer not to make that change later; right now I just want to get the tests committed.

**Comment by junwang-noaa** [comment]
I agree. When we have results on performance analysis to optimize the computer resources, we can change PETs to reduce time.
